web-app/js/portal/details/LayerControlPanel.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2012 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.details');
+
+Portal.details.LayerControlPanel = Ext.extend(Ext.Container, {
+
+    initComponent: function() {
+        this.opacitySlider = new Portal.common.LayerOpacitySliderFixed({
+            layer: new OpenLayers.Layer("Dummy Layer"),
+            keyIncrement: 10,
+            increment: 5,
+            minValue: 20, // minimum visibility for the current layer is 20%
+            maxValue: 100,
+            aggressive: true,
+            width: 175,
+            isFormField: true,
+            inverse: false,
+            fieldLabel: OpenLayers.i18n('Opacity'),
+            plugins: new GeoExt.LayerOpacitySliderTip({
+                template: '<div class="opacitySlider" >Opacity: {opacity}%</div>'
+            })
+        });
+
+        // Put in container so that we see the label.
+        this.opacitySliderContainer = new Ext.Panel({
+            layout: 'form',
+            items: [ this.opacitySlider ]
+        });
+
+        this.layerVisibilityCheckbox = new Ext.form.Checkbox({
+            value: true,
+            boxLabel: OpenLayers.i18n('showMapLayer'),
+            checked: true,
+            listeners: {
+                scope: this,
+                check: this._visibilityButtonChecked
+            }
+        });
+
+        this.zoomToLayer = new Ext.ux.Hyperlink({
```

Comment:
Then I'm not sure about it. However also @pmbohm said something about it which I didn't quite understand at the time, but said we probably should get rid of those too.

We're not moving to Ext4 any time soon, so I guess it's fine anyway. However it'd be good to start removing out dependence on `Ext.ux` because it locks us down a bit tighter to specific Ext versions, making the upgrade more difficult if it ever takes place.